### DISCUSSION TITLE
DDoS-Guard - Simpcity.su

### DISCUSSION THREAD

**Discussion by poukoipas974** [jbsparrow/CyberDropDownloader#92] [General]
Hi,
Now Simpcity.su has implemented a ddos-guard. Would it be possible to work around this?
All download failed with DDoS-Guard errors.

**Comment by jbsparrow** [comment]
No. They will disable it when they stop getting DDOSed.

**Reply by poukoipas974** [reply]
Is there anything I can do to speed this up?

**Reply by jbsparrow** [reply]
Don't try to scrape the site for a while. It will reduce traffic.
